### DISCUSSION TITLE
✨ Heimdall

### DISCUSSION THREAD

**Discussion by JigSawFr** [runtipi/runtipi-appstore#2240] [Requests]
### Description

As the name suggests Heimdall Application Dashboard is a dashboard for all your web applications. It doesn't need to be limited to applications though, you can add links to anything you like.

Heimdall is an elegant solution to organise all your web applications. It’s dedicated to this purpose so you won’t lose your links in a sea of bookmarks.

Why not use it as your browser start page? It even has the ability to include a search bar using either Google, Bing or DuckDuckGo.

### Official Website

https://heimdall.site/

### Documentation link

https://github.com/linuxserver/Heimdall

### Last application release & date

2.5.8 (11/11/2023)

### Application license

MIT license

### Source code repository link

https://github.com/linuxserver/Heimdall

### Docker image link

https://hub.docker.com/r/linuxserver/heimdall/

### Other's

Originally requested on Tipi's Discord.

### Please confirm the following

- [X] I confirm that I've put the app name in the title of the discussion
- [X] I confirm that the application is actively maintained.
- [X] I confirm that the docker image of the application is actively maintained.
- [X] I confirm that I have indeed checked that no similar request for this application already exists

### Gathering crowds

Hey folks!

Please upvote ⬆️ this discussion to show your interest in this request! 

Thanks ⛺

**Comment by steveiliop56** [comment]
added
